server/task_scheduler.go

```diff
@@ -17,6 +17,7 @@ import (
 
 const (
 	taskSchedulerInterval = time.Duration(1) * time.Second
+	taskProgressInterval  = time.Duration(5) * time.Second
```

Comment:
Prefer `taskProgressUpdateInterval`.
IMO, we can set it to a more frequent value like one second.